src/Felix/Equiv.agda

```diff
@@ -32,13 +32,17 @@ record Equivalent q {obj : Set o} (_⇨_ : obj → obj → Set ℓ)
   module ≈-Reasoning {a b} where
     open SetoidR (≈setoid a b) public
 
-  infixr 9 _•_
-  _•_ : {f g h : a ⇨ b} (g≈h : g ≈ h) (f≈g : f ≈ g) → f ≈ h
-  g≈h • f≈g = trans f≈g g≈h
+    infix 2 ⟺
+    ⟺ : {f g : a ⇨ b} → f ≈ g → g ≈ f
+    ⟺ = sym
```

Comment:
What motivated introducing and using this synonym for `sym`? Is it related to the other changes in this PR?

Reply:
Ok, this pointed out that I didn't spell out my bigger questions here. Right now the we open `Equiv` publicly, under the `Equivalent`. Using `sym` (and `refl`, `trans`) directly will have the problems we are seeing. I think that we shouldn't reexport those functions publicly, but I didn't want to change that without discussing.

That's why I introduced alias for `sym` which comes from `≈-Reasoning` module and can have `Equivalent` instance passed explicitly.

Alias itself and its fixity declarations come from agda- categories and I don't know the answer to the fixity question.

Another approach would be to reexport familiar names from the reasoning module instead.

Reply:
Thank you for the context. I had not noticed that you moved `_;_` and `_•_` under `≈-Reasoning`. Nice trick!

I agree with you about reexporting `refl`, `sym`, and `trans` from `≈-Reasoning` instead of from `Equivalent`. Thanks for the suggestion.